app/poros/nytimes_results.rb

```diff
@@ -3,64 +3,49 @@
 
 class NytimesResults
   def get_obituaries
-    json = NytimesService.new.get_obituaries
-
-    json[:response][:docs].map do |obituary_data|
-
-      Obituary.new(format_obituary_data(obituary_data))
```

Comment:
Refactored all of these to a helper method because it was repeated in all of the methods.

Reply:
Nice work Jack!